rcongui/src/components/RconSettings/rconSettings.js

```diff
@@ -39,6 +42,14 @@ class RconSettings extends React.Component {
             }))
     }
 
+    async getBannedUnblacklistedPlayers() {
```

Comment:
Does that belong in the UI? It's something to fix the sate right? Maybe it's best done at startup or something of the sort?

Reply:
It's for server admins to unban players who were previously blacklisted and got banned, but are now unblacklisted and should hence be unbanned. This happens automatically on the game server instance associated with the rcon tool instance, but not on other servers which share the same DB. We could trigger it on start up as well, but it might still be useful to also have a manual action in case the rcon tool doesn't get restarted often?